src/sorcha/ephemeris/simulation_driver.py

```diff
@@ -0,0 +1,98 @@
+import spiceypy as spice
+from sorcha.ephemeris.simulation_setup import *
+from sorcha.ephemeris.simulation_constants import *
+from collections import defaultdict
+from os import path
+import csv
+from sorcha.ephemeris.simulation_geometry import *
+from sorcha.ephemeris.simulation_parsing import *
+import numpy as np
+from sorcha.utilities.dataUtilitiesForTests import get_data_out_filepath
+from sorcha.modules.PPReadPointingDatabase import PPReadPointingDatabase
+
+out_csv_path = get_data_out_filepath("ephemeris_output.csv")
+
+
+def create_ephemeris(args, configs):
+    ang_fov = configs["ar_ang_fov"]
+    buffer = configs["ar_fov_buffer"]
+    picket_interval = configs["ar_picket"]
+    obsCode = configs["ar_obs_code"]
+    nside = 2 ** configs["ar_healpix_order"]
+    first = 1  # Try to get away from this
+
+    t_picket = 2460000.5
```

Comment:
Should this be hardcoded? Doesn't have to fixed now, more so if we should have a ticket to addres this later.